tools/eksDistroBuildToolingOpsTools/cmd/eksGoTool/cmd/backportIssue.go

```diff
@@ -0,0 +1,32 @@
+package cmd
+
+import (
+	prmanager "github.com/aws/eks-distro-build-tooling/tools/eksDistroBuildToolingOpsTools/pkg/prManager"
+	"github.com/spf13/cobra"
+)
+
+const ()
+
+var (
+	backportIssue = &cobra.Command{
+		Use:   "backport",
```

Comment:
SHould we use subcommands here, like "backport issue" or "backport pr" since we will likely use this command elsewhere?